### DISCUSSION TITLE
Support for macOS Monterey?

### DISCUSSION THREAD

**Discussion by ghost** [rxhanson/Rectangle#525] [General]
Hi, will Rectangle function in macOS Monterey? @rxhanson

**Comment by rxhanson** [comment]
I typically don't do any testing in macOS betas. In the past Apple has broken things, I've spent time looking into it, and then Apple fixes things. It's best just to wait. If anyone else tests on a beta and fixes bugs, then I wouldn't mind creating a release with any bug fixes.

When the official release comes out, I will fix any bugs due to the macOS upgrade.

Of course, with any macOS upgrade, there's no 100% guarantee that Apple won't irreversibly break things for a 3rd party app, but it's a really safe bet that Apple wouldn't do that. I would be surprised if Rectangle didn't just work out of the box on Monterey, anyways. 

In short, yes, my plans are to support every new version of macOS.

**Comment by fibonacid** [comment]
Doesn't work for me

**Reply by fibonacid** [reply]
Updating to v0.49 fixed it.
